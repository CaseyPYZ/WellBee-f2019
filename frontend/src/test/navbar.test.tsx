
import React from 'react';
import '../setup.test';
import { shallow } from 'enzyme';
import NavBarComponent from '../components/navbar/navbar.component';

describe('Navbar', () => {
  it('Navbar Component should render', () => {
    const component = shallow(<NavBarComponent />);
    expect(component).toMatchSnapshot();
  });
});
