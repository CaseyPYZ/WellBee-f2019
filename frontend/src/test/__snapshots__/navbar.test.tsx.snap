// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Navbar Component should render 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark static-top"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      <img
        alt=""
        src="wellbee.logo.png"
      />
    </a>
    <ul
      className="navbar-nav ml-auto"
    >
      <li
        className="nav-item active"
      >
        <a
          className="nav-link"
          href="/"
        >
          Home
          <span
            className="sr-only"
          >
            (current)
          </span>
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/about"
        >
          About
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/contact"
        >
          Contact Us
        </a>
      </li>
    </ul>
    <div
      className="collapse navbar-collapse"
      id="navbarResponsive"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="/login"
          >
            Login
          </a>
        </li>
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="/signup"
          >
            Signup
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
